examples/spark/README.md

```diff
@@ -228,9 +255,21 @@ zeppelin-controller-ja09s   1/1       Running   0          53s
 Now you have two choices, depending on your predilections. You can do something
 graphical with the Spark cluster, or you can stay in the CLI.
 
+For both choices, we will be working with this Python snippet:
```

Comment:
The pyspark console example needs the code raw, and the zeppelin example needs the `%pyspark` hint.  It's a repetition vs ease of use thing. I'll rewrite it to make it more clear why we're repeating.